src/Composer.vala

```diff
@@ -561,11 +564,17 @@ public class Mail.Composer : Hdy.ApplicationWindow {
         }
 
         if (content_to_quote != null) {
-            string message_content;
-
             if (type == Type.DRAFT) {
                 ancestor_message_info = info;
-                message_content = content_to_quote;
+
+                if (content_to_quote.contains ("elementary-message-body")) {
```

Comment:
I can also test this myself if you aren't sure

Reply:
In theory it should work because `querySelector` always returns the first occurrence oft the element but you're right it might be better to remove the ids when sending.

Reply:
The id attribute is now always removed when sending